src/lsp/sql_ast/sql_ast.ml

```diff
@@ -0,0 +1,565 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+open Cobol_common
+(**************************************************************************)
+(*                              AST                                       *)
+(**************************************************************************)
+type sql_var = string with_loc 
+[@@deriving ord]
+type cobol_var_id = string with_loc
+[@@deriving ord]
+
+type cobol_var =
+| NotNull of cobol_var_id
+| NullIndicator of cobol_var_id * cobol_var_id
+[@@deriving ord]
+
+type variable = 
+  | SqlVar of sql_var
+  | CobolVar of cobol_var
+  [@@deriving ord]
+  
+
+type literal =
+  | LiteralVar of variable
+  | LiteralNum of string with_loc
+  | LiteralStr of string with_loc
+  | LiteralDot of string with_loc list
+  [@@deriving ord]
+
+
+type sql_token = 
+  | Sql_instr of string
```

Comment:
Just for consistency: since you used camel case in all other types, you should do that here as well